Realm/RLMSyncSession.h

```diff
@@ -56,6 +106,25 @@ NS_ASSUME_NONNULL_BEGIN
 /// session.
 - (nullable RLMSyncConfiguration *)configuration;
 
-NS_ASSUME_NONNULL_END
+/**
+ Register a progress notification block. Multiple blocks can be registered with the same session at once. All blocks
+ will be run on the same thread that they were registered on. The progress notification block will always be called
```

Comment:
"All blocks will be run on the same thread that they were registered on" -> "Each block will be invoked from the runloop of the thread on which it was registered". The existing text threw me off because it starts by saying "all blocks will be run on the same thread".